### PR TITLE
Fix/project created tracking

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5287
| License          | BSD 3-Clause

## Description

Add a page tracking event, after a public cloud project is created, with the project id and voucher code as page parameters

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/creating/routing.js [hunk 1]
```diff
@@ -17,10 +17,21 @@ export default /* @ngInject */ ($stateProvider) => {
...
-      onProjectDelivered: /* @ngInject */ ($state) => (projectId) =>
-        $state.go('pci.projects.project', {
+      onProjectDelivered: /* @ngInject */ ($state, atInternet, voucherCode) => (
+        projectId,
+      ) => {
+        const trackingData = {
+          name: 'public-cloud::pci::projects::created',
           projectId,
-        }),
+        };
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/creating/routing.js:26]: **suggestion**: Another way to write it that avoids a `if` could be : 
```suggestion
(...voucherCode ? { voucherCode} : {})
        };
```

Comment (ganeshkumar1989) [reply packages/manager/modules/pci/src/projects/creating/routing.js]: This is done :)

**Comment by antleblanc** [comment]
@ganeshkumar1989 Could you please rebase it on `master` branch?
Thank you!

**Comment by antleblanc** [comment]
Rebase in progress in order to rewrite commit history.

**Comment by antleblanc** [comment]
Rebase on `develop` done. (only commit message has been rewritten)

**Pull request merged**